server/app.js

```diff
@@ -45,6 +46,7 @@ app.use((req, res, next) => {
 app.use("/auth", authRouter);
 app.use("/users", userRouter);
 app.use("/profile", profileRouter);
+app.use("/availability", availabilityRouter);
```

Comment:
Looking at the controller work - `/availability/schedule` probably would make more sense. I think the direction we'll go is if they create an account as a pet sitter it will automatically create the availability document, and if they convert to a pet sitter too.